ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json

```diff
@@ -20,6 +20,7 @@
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BOCaseStoppedAwaitRedec", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BORedecNotificationSent", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BOSotGenerated", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "Dormant", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
```

Comment:
Do we only want to add it for these two roles?

Reply:
Possibly. Not sure this PR will be the one to deploy the new state though. I just needed it for this PR to work